understanding/non-text-contrast.html

```diff
@@ -108,395 +102,279 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
-               <p>The intent of this Success Criterion is to ensure that user interface components (i.e.,
-                  controls) and meaningful graphics are distinguishable by people with moderately low
-                  vision. The requirements and rationale are similar to those for large text in <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>. Note that this requirement does not apply to <em>inactive</em> user interface components.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、アクティブなユーザインタフェース コンポーネント (つまり、コントロール) 及び意味のあるグラフィックが中程度のロービジョンの人によって識別できるようすることである。この必須要件の根拠は、<a href="contrast-minimum">1.4.3 コントラスト (最低限)</a> の大きな文字のものと同様である。この要件は、<em>非アクティブな</em>ユーザインタフェース コンポーネントには適用されないことに注意されたい。
                </p>
-               <p>Low contrast controls are more difficult to perceive, and may be completely missed
-                  by people with a visual impairment. Similarly, if a graphic is needed to understand
-                  the content or functionality of the webpage then it should be perceivable by people
-                  with low vision or other impairments without the need for contrast-enhancing assistive
-                  technology.
+               <p>低いコントラストのコントロールは知覚するのが難しく、かつ視覚障害のある人に完全に見逃される可能性がある。同様に、ウェブページのコンテンツ又は機能を理解するためにグラフィックが必要とされる場合、コントラストを強化する支援技術の必要なくロービジョン又は他の障害のある人に知覚されるべきある。
                </p>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      				
-                     <p>The 3:1 contrast ratios referenced in this Success Criterion is intended to be treated
-                        as threshold values. When comparing the computed contrast ratio to the Success Criterion
-                        ratio, the computed values should not be rounded (e.g. 2.999:1 would not meet the
-                        3:1 threshold).
+                     <p>この達成基準で言及されている 3:1 のコントラスト比は、閾値として扱われることを意図している。算出されたコントラスト比を達成基準のコントラスト比と比較する場合、算出された値は丸めるべきではない（例えば、 2.999:1 は 3:1 という閾値を満たさない）。
                      </p>
                      			
                   </div>
                </div>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      				
-                     <p>Because authors do not have control over user settings for font smoothing and anti-aliasing,
-                        when evaluating this
-                        					 Success Criterion, refer to the colors obtained from the user agent, or the
-                        underlying
-                        					 markup and stylesheets, rather than the non-text elements as presented on screen.
+                     <p>フォントスムージング及びアンチエイリアシングのユーザー設定をコンテンツ制作者はコントロールできないので、この達成基準を評価するときは、画面に表示される非テキスト要素よりも、ユーザエージェントで取得できる、つまり基礎となるマークアップ及びスタイルシートから得られる、色を参照すること。
```

Comment:
https://github.com/waic/wcag22/pull/361/files#r1957006667 と原文が微妙に違うのはどうなんだと思いますが。

```suggestion
                     <p>フォントスムージング及びアンチエイリアシングのユーザ設定をコンテンツ制作者は制御できないので、この達成基準を評価するときは、画面に表示される非テキスト要素ではなく、ユーザエージェントから取得した色、つまり基礎となるマークアップ及びスタイルシートから取得した色を参照すること。
```